src/index.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import {App} from "./components/app/app.jsx";
+
+const unit = () => {
```

Comment:
Небольшая опечатка, в задании написано `init`.
```suggestion
const init = () => {
```